### DISCUSSION TITLE
att-ipv6 for automating wan ip between the at&t modem and udm pro?

### DISCUSSION THREAD

**Discussion by teaglebuilt** [unifi-utilities/unifi-common#538] [Q&A]
Just curious if that script will automate adleast udm pro's part of obtaining the wan ip from the at&t modem on passthrough mode?

Im thinking about ways to do this.

**Comment by boostchicken** [comment]
Should just be setting prefix delegation to 60.  Also get that modem out of thre hot garbage :(
